### DISCUSSION TITLE
sSNV QC

### DISCUSSION THREAD

**Discussion by graceooh** [uclahs-cds/pipeline-call-sSNV#176] [Q&A]
Hey @maotian06,

I was wondering if you had any experience in getting or plotting any of the following:
1) number of sSNVs 2) overlaps/differences between callers 3) sSNV numbers per chromosome 4) kataegis 5) VAF distribution between callers. It would be wonderful if you could share your scripts or experience.
Thanks!

Jieun

**Comment by maotian06** [comment]
1. Number of sSNVs：You should be able to get this from VCFs.
2. Overlaps: you can used my scripts: `/hot/software/pipeline/pipeline-call-sSNV/Nextflow/development/6.0.0-rc.1/maotian-add-muse-workflow/compare-vcf/bash.sh`, it will output an venn plot for each sample and a comparsion result. You might need to adjust the script to suit your data.
<img width="300" alt="image" src="https://user-images.githubusercontent.com/47042983/234077856-0abb9af0-87e9-4a8f-b038-4e48c3930a30.png">

3.  sSNV numbers per chromosome: You should get from VCF.
4. kataegis: Check the [SeqKat package] (https://github.com/uclahs-cds/public-R-SeqKat)
5. VAF distribution: I think @WuSelina  and @sorelfitzgibbon have some scripts?

**Reply by graceooh** [reply]
Ok I don't know why I wrote 1&3 😂 Thank you!

**Comment by sorelfitzgibbon** [comment]
Here’s a link to @WuSelina's PR for  “creating a stripplot of average adjusted VAFs for sSNVs called by each algorithm”

https://github.com/uclahs-cds/project-disease-KidneyTumor-GHLR-000108-SubclonalArchitecture/pull/3
